src/browser.coffee

```diff
@@ -26,10 +26,8 @@ return unless window?
 # Ported from https://developer.mozilla.org/en-US/docs/DOM/window.btoa
 if btoa? and JSON? and unescape? and encodeURIComponent?
```

Comment:
I wonder if this check is still needed. Also, have you tested if the test and fix from commit b2d6997 still works after your changes?